api/classes/database.js

```diff
@@ -0,0 +1,16 @@
+const config = require('../../knexfile')['development'];
+
+// reset database at each startup
+//try {
```

Comment:
frankly, i think i should just delete this, it's from a demo i did for endpoints and doesn't really apply here- agree?